.github/workflows/go-testing-ci.yml

```diff
@@ -83,5 +83,9 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           files: ./coverage.txt
-          flags: gointegrationtests
-
+          flags: gointegrationtests   
+      - name: Start Mesheryctl System
```

Comment:
This step will come after creating k8s cluster why did I said this was to see if Meshery server is running and then we run the test will it then pass or not add it after line 78 and under run script specify this 
`cd mesheryctl; make
./mesheryctl system start`

Reply:
@suhail34 
But 
` - name: Run coverage
        run: go test -run Integration ./... -race -coverprofile=coverage.txt -covermode=atomic -timeout 40m`
runs the integration tests already

Reply:
`flags: gointegrationtests   
      - name: Start Mesheryctl System
        run: |
          cd mesheryctl; make; ./mesheryctl system start `

Reply:
or 
`

          flags: gointegrationtests 
          cd mesheryctl; make; ./mesheryctl system start
`

Reply:
> 

Yes